src/components/DuosHeader.js

```diff
@@ -8,4 +1,111 @@
+import {Component, Fragment, useState} from "react";
+import {a, button, div, h, hr, img, li, nav, small, span, ul} from "react-hyperscript-helpers";
+import Drawer from "@material-ui/core/Drawer";
+import Hidden from "@material-ui/core/Hidden";
+import {IconButton, List, ListItem, Menu, MenuItem} from "@material-ui/core";
+import { withStyles } from "@material-ui/core/styles";
+import MenuIcon from "@material-ui/icons/Menu";
+import {Link, withRouter} from "react-router-dom";
+import {Storage} from "../libs/storage";
+import {SupportRequestModal} from "./modals/SupportRequestModal";
+import "./DuosHeader.css";
 import {NavigationUtils} from "../libs/utils";
+import { Styles } from "../libs/theme";
+
+const styles = {
+  drawerPaper: {
+    backgroundColor: '#00243c',
+    color: 'white',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=6963768)
```suggestion
    fontFamily: "Montserrat"
```